src/test/mgr/test_progress.py

```diff
@@ -130,20 +121,21 @@ class TestModule(object):
     # Testing Module Class
     
     def setup(self):
-        # Creating the class and Mocking
-        # attributes for testing
-        self.test_class = module
-        self.test_class._module = Mock()
-        self.test_class.PgRecoveryEvent.pg_update = Mock()
-        self.test_module = module.Module
-        self.log = Mock()
-        self.get = Mock()
-        self._complete = Mock()
-        self._events = {}
+        # Creating the class and Mocking a
+        # bunch of attributes for testing
+
+        module.PgRecoveryEvent.pg_update = Mock()
+        self.test_event = module.Module() # so we can see if an event gets created
```

Comment:
since this is an instance of Module, perhaps calling it test_module instead of test_event would be clearer